### DISCUSSION TITLE
Is it a bug or my code problem?

### DISCUSSION THREAD

**Discussion by wlxmhls** [raysan5/raylib#2565] [Q&A] [answered]
Hi, I create a window using raylib. When I drag the window's title below the Microsoft Windows taskbar (means hiding part of the raylib window), and then drag it above the taskbar (means showing the whole part of raylib window), I see the raylib window seems corrupted.
I take a screenshot.

![1](https://user-images.githubusercontent.com/5212346/177229123-86c0c379-8964-466d-a813-ca7352cc5ee6.png)

What is the reason?

The below is my code:

```
#include <raylib.h>

#define BOARD_W  420
#define BOARD_H  250

const char msgText[128] = "This is a text\nfor typing effect show\nPlease enjoy it\n\n";

static void DrawBoarder(int x, int y, int w, int h, Color col);


int main (void)
{
    const int screenWidth = 800;
    const int screenHeight = 450;

    InitWindow(screenWidth, screenHeight, "raylib example");

    SetTargetFPS(60);

    while (!WindowShouldClose())
    {

        BeginDrawing();
            ClearBackground(RAYWHITE);

            DrawRectangle(0, 0, screenWidth, 20, BLACK);
            DrawText("Press SPACE for finishing message OR replaying message.", 0, 0, 20, WHITE);

            DrawBoarder((screenWidth-BOARD_W)/2, (screenHeight-BOARD_H)/2, BOARD_W, BOARD_H, BLACK);
            DrawBoarder((screenWidth-BOARD_W)/2+1, (screenHeight-BOARD_H)/2+1, BOARD_W-2, BOARD_H-2, GRAY);
            DrawBoarder((screenWidth-BOARD_W)/2+5, (screenHeight-BOARD_H)/2+5, BOARD_W-10, BOARD_H-10, BLACK);

        EndDrawing();
    }

    CloseWindow();
    return 0;
}

void DrawBoarder(int x, int y, int w, int h, Color col)
{
    int r = 10;
    DrawCircle(x+r, y+r, r, col);
    DrawCircle(x+w-r, y+r, r, col);
    DrawCircle(x+r, y+h-r, r, col);
    DrawCircle(x+w-r, y+h-r, r, col);

    DrawRectangle(x, y+r, w, h-2*r, col);
    DrawRectangle(x+r, y, w-2*r, h, col);
}
```

**Comment by raysan5** [comment, marked as answer]
@wlxmhls I'm afraid I can't reproduce the issue. Just tried this code on my Windows 10 64bit and it works as expected, no framebuffer data "corruption". Looking at the screenshot it seems issue could be related to window refreshing, maybe related to graphics driver?

**Reply by wlxmhls** [reply]
Thanks! I am not sure if it's related to my Windows 7. But the other windows/dialogs do not have this effect.

**Comment by wlxmhls** [comment]
Maybe it is my Windows 7 problem. It is okay on my Linux box.

**Reply by raysan5** [reply]
Yeah, it's an old OS, that can probably be the issue
